app/routes/movies.$id.jsx

```diff
@@ -1,19 +1,32 @@
 import { json } from "@remix-run/node";
 import { useLoaderData } from "@remix-run/react";
+import styles from "../styles/movie-details.css";
 
 export const loader = async ({ params }) => {
   const res = await fetch(`https://freetestapi.com/api/v1/movies/${params.id}`);
   const movie = await res.json();
   return json(movie);
 };
```

Comment:
We're not living in an ideal world and errors will happen many times, so you need to handle them here
try/catch or then/catch
and add [error boundaries](https://remix.run/docs/en/main/route/error-boundary)